examples/multiple_secrets/k8s-eks/Identity.yaml

```diff
@@ -13,6 +13,7 @@ kind: ClusterSPIFFEID
 metadata:
   name: example
 spec:
+  className: "vsecm"
```

Comment:
This `className` is important because that’s how SPIRE manages its internal `ClusterSPIFFEID`s too.